src/helpers/buildWalletSections.js

```diff
@@ -100,7 +100,7 @@ const addEth = section => {
           amount: '0',
           display: '$0',
         },
-        change: `${relative_change_24h.toFixed(2)}%`,
+        change: `${relative_change_24h?.toFixed(2)}%`,
```

Comment:
This fixes the crash but we might show "%" if it's falsey.

Unless I'm wrong and we're checking that somewhere else, would be better if we check and return an empty string instead.